alerts.md

```diff
@@ -294,15 +294,17 @@ Cluster health alerts occur when the cluster health score changes by a specific
 * Out of Memory Pods
 * Failed Jobs
 
-This alert only uses Slack and Microsoft Teams, so it requires the `global...WebhookUrl` field, or setting the `...WebhookUrl` field for either platform for the alert.
-
```

Comment:
@bstuder99 I was wondering whether we should just drop this line or include a mention like "From v1.101 onwards, email recipients are supported alongside slack & ms teams..."

Reply:
We generally don't mention versioning when it comes to when features are available unless it requires Enterprise, or is related to an API (usually for deprecation/retirement). We generally assume users have the most recent version (even though some choose not to upgrade). I'd actually like to get @kwombach12's take on this